### PR TITLE
[mod_users_latest]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_users_latest
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_users_latest/mod_users_latest.xml [hunk 1]
```diff
@@ -25,58 +25,61 @@
...
 					name="layout"
 					type="modulelayout"
 					label="JFIELD_ALT_LAYOUT_LABEL"
-					description="JFIELD_ALT_MODULE_LAYOUT_DESC" />
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC"
+				/>
 
 				<field
 					name="moduleclass_sfx"
 					type="textarea" rows="3"
```
Comment (andrepereiradasilva) [inline modules/mod_users_latest/mod_users_latest.xml]: shoulfn't rows be moved down?

File: modules/mod_users_latest/mod_users_latest.xml [hunk 2]
```diff
@@ -25,58 +25,61 @@
...
+					default="900"
+				/>
+
 				<field
 					name="cachemode"
 					type="hidden"
 					default="static">
-					<option
-						value="static"></option>
+					<option value="static"></option>
```
Comment (andrepereiradasilva) [inline modules/mod_users_latest/mod_users_latest.xml]: move the > to the next line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on c8cc0e49ce192947eb67e8d7c8c6d4240d895bff<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10596">issues.joomla.org/joomla-cms/10596</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on c8cc0e49ce192947eb67e8d7c8c6d4240d895bff<br /><br />on review - Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10596">issues.joomla.org/joomla-cms/10596</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10596">issues.joomla.org/joomla-cms/10596</a>.</sub>

**Pull request closed without merging**